components/centraldashboard/license_info.csv

```diff
@@ -0,0 +1,1287 @@
+@babel/plugin-transform-shorthand-properties@7.2.0,https://github.com/babel/babel/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/babel/babel/master/LICENSE
+@babel/plugin-transform-spread@7.6.2,https://github.com/babel/babel/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/babel/babel/master/LICENSE
+@babel/plugin-transform-sticky-regex@7.2.0,https://github.com/babel/babel/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/babel/babel/master/LICENSE
+@babel/plugin-transform-template-literals@7.4.4,https://github.com/babel/babel/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/babel/babel/master/LICENSE
+@babel/plugin-transform-typeof-symbol@7.2.0,https://github.com/babel/babel/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/babel/babel/master/LICENSE
+@babel/plugin-transform-unicode-regex@7.6.2,https://github.com/babel/babel/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/babel/babel/master/LICENSE
+@babel/polyfill@7.6.0,https://github.com/babel/babel/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/babel/babel/master/LICENSE
+@babel/preset-env@7.6.2,https://github.com/babel/babel/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/babel/babel/master/LICENSE
+@babel/runtime@7.6.2,https://github.com/babel/babel/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/babel/babel/master/LICENSE
+@babel/template@7.6.0,https://github.com/babel/babel/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/babel/babel/master/LICENSE
+@babel/traverse@7.6.2,https://github.com/babel/babel/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/babel/babel/master/LICENSE
+@babel/types@7.6.1,https://github.com/babel/babel/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/babel/babel/master/LICENSE
+@google-cloud/monitoring@0.7.1,https://github.com/googleapis/nodejs-monitoring/tree/master/LICENSE,Apache-2.0,https://raw.githubusercontent.com/googleapis/nodejs-monitoring/master/LICENSE
+@google-cloud/promisify@0.4.0,https://github.com/googleapis/nodejs-promisify/tree/master/LICENSE,Apache-2.0,https://raw.githubusercontent.com/googleapis/nodejs-promisify/master/LICENSE
+@grpc/grpc-js@0.3.6,https://github.com/grpc/grpc-node/tree/master/LICENSE,Apache-2.0,https://raw.githubusercontent.com/grpc/grpc-node/master/LICENSE
+@grpc/proto-loader@0.4.0,https://github.com/grpc/grpc-node/tree/master/README.md,Apache-2.0,https://raw.githubusercontent.com/grpc/grpc-node/master/README.md
+@kubernetes/client-node@0.8.2,https://github.com/kubernetes-client/javascript/tree/master/LICENSE,Apache-2.0,https://raw.githubusercontent.com/kubernetes-client/javascript/master/LICENSE
+@polymer/app-layout@3.1.0,https://github.com/PolymerElements/app-layout/tree/master/README.md,BSD-3-Clause,https://raw.githubusercontent.com/PolymerElements/app-layout/master/README.md
```

Comment:
I don't see license content in the README, but
https://github.com/PolymerElements/app-layout/blob/master/package.json has the license title only.

Is it better to point to the package.json?

Reply:
Fixed with global polymer license